report/sections/secondary-groups.tex

```diff
@@ -1,3 +1,18 @@
 \section{Secondary Groups}
 
-\todo{TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group}
+%TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group
+Our Secondary groups were Computer Vision (CV) and Emotion Synthesis (ES). Our task in the final pipeline was to read the input given by CV group and send the output to ES group.
+
+\subsection{Computer Vision Group}
+%Format read from CV Group
+After training our models with the Bosphorus database we need to predict an emotion by reading input given from the CV group. They are giving input in a numpy array containing $25$ landmarks. PCA was used for dimensionality reduction before training our model, so the same PCA model is used on the given input for reducing the number of features.
```

Comment:
``They are giving input in a numpy array containing $25$ landmarks``. They are not given this, rather they create it and give it to us (sentence needs to be rephrased).